README.md

```diff
@@ -117,6 +117,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Yakamoz Technology](http://www.yakamoztech.com/) | Kamaladi, Kathmandu | We believe in building products that are simple and intuitive
 - [Yomari Information Services Incorporated](http://yomari.com/) | Kumaripati, Lalitpur | Analytics, business intelligence (BI), and data warehouse consultants
 - [YoungInnovations](http://younginnovations.com.np/) | Kumaripati, Lalitpur | We create innovative and powerful solutions to global development problems
+- [Yarsa Labs](https://yarsa.io/) | Zero Kilometer, Pokhara 33700 | Building and distributing high quality software applications for mobile and the web.
```

Comment:
Thank you for the pull request. 

- The list is in alphabetical order, you'll have to move `Yarsa Labs` after `Yakamoz Technology`.
- Also, can you remove `33700`. 
- Please remove the full stop at the end of the description too.